src/OneExplorer/ConfigObject.ts

```diff
@@ -225,6 +231,20 @@ export class ConfigObj {
       })
     });
 
+    locatorRunner.register({
+      artifactAttr: {
+        ext: '.json',
+        icon: new vscode.ThemeIcon('graph'),
+        openViewType: 'one.editor.jsonTracer',
+        canHide: true
+      },
+      locator: new Locator(
+          (value: string) => {
+            return LocatorRunner.searchWithCommandOption(value, '--save-chrome-trace', '.json');
```

Comment:
`--save-chrome-trace` is backend-specific option and I guess this is OK for quick fix because we already have many backend specific terms in this file (`tracealloc.json`, `tvn` (on L217, L230 etc), `tv2m`,... ) 😹 
Maybe after release, we should eventually remove the these backend-specific terms from this file.

Reply:
Sure ;-D Actually I was working on that and tried backend-refactoring before. I will raise up its priority after this.